streams-runtimes/streams-runtime-local/src/main/java/org/apache/streams/local/tasks/StreamsPersistWriterTask.java

```diff
@@ -146,7 +146,7 @@ public void run() {
             try {
                 Thread.sleep(5000);
             } catch (InterruptedException e) {
-                LOGGER.error("Sleep interrupted: {}", e);
+                //This is the expected behavior when this thread is killed
```

Comment:
Higher level question: Why are we sleeping like this in the finally block?  Is there some work we think is still going on?